packages/kbn-optimizer/limits.yml

```diff
@@ -122,8 +122,8 @@ pageLoadAssetSize:
   observabilityShared: 111036
   osquery: 107090
   painlessLab: 179748
-  presentationPanel: 11468
-  presentationUtil: 25000
+  presentationPanel: 26530
```

Comment:
Mind if we roll these back?  They were recently updated, I believe the preference was to keep it close to the current bundle size.